rules/falco_rules.yaml

```diff
@@ -445,7 +445,8 @@
     (fd.filename in (shell_config_filenames) or
      fd.name in (shell_config_files) or
      fd.directory in (shell_config_directories)) and
-    not proc.name in (shell_binaries)
+    not proc.name in (shell_binaries) and
```

Comment:
Correct indentation pls :)

Reply:
you mean the indentation of the whole condition? I believe this line is properly indented within this rule: the "inner" block ends at the `))` in line 447

Reply:
The indentation looks right to me

Reply:
You right @fcoelho ! Sorry, resolving